Source/FolioReaderCenter.swift

```diff
@@ -1281,7 +1272,7 @@ open class FolioReaderCenter: UIViewController, UICollectionViewDelegate, UIColl
         // Perform the page after a short delay as the collection view hasn't completed it's transition if this method is called (the index paths aren't right during fast scrolls).
         delay(0.2, closure: { [weak self] in
             if (self?.readerConfig.scrollDirection == .horizontalWithVerticalContent),
-                let cell = ((scrollView.superview as? UIWebView)?.delegate as? FolioReaderPage) {
+                let cell = ((scrollView.superview as? WKWebView)?.navigationDelegate as? FolioReaderPage) {
                 let currentIndexPathRow = cell.pageNumber - 1
```

Comment:
@ThakarRajesh I have already fixed the UIWebView reference - take a look at commit 49076e400c35bcfd2f0c2adbf27bde21f9755e84